### PR TITLE
Update Readme.md at root of repository

### LINKED ISSUE #102: Update Readme.md at root of repository
Lets remove existing stuff and put details on how to use this repository.

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #102

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
-# test-configs
-gsoc 24 project where will store configs for diff.groovy , https://github.com/orgs/checkstyle/projects/8
+# Checkstyle Test Configurations Generator
```
Comment (romani) [inline README.md:1]: Please put this content to contribution.md

Example https://github.com/checkstyle/checkstyle/blob/master/.github/CONTRIBUTING.md

But in Readme put details on how to use configs. 99% of usage of this repository will be by people who has 0 interest in java code of this repository. All user care is just how to use configs of this repository.

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: README.md (hunk 1 above)
Comment (romani) [reply README.md:1]: Still not done

Comment (relentless-pursuit) [reply README.md:1]: Is the README right? I am preparing for CONTRIBUTION.md.

**Review by romani — APPROVED** [review]

**Pull request merged**